### PR TITLE
[build] create workflow to PR changes to browser versions

### PR DESCRIPTION
**Pull request by titusfortner**
This is set to run daily and create a PR with any changes.

The PR will look like this: https://github.com/titusfortner/selenium/pull/54

(I'm not sure why it was posted as me and not the CI bot, but it was posted by this action: https://github.com/titusfortner/selenium/actions/runs/7585149002)

I did this with patch files in case we want to use this approach for other things in the future.

### CODE REVIEW

File: .github/workflows/pin-browsers.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      name: Pin Browsers
+      cache-key: pin-browsers
+      run: bazel run //scripts:pinned_browsers
+
+  pull-request:
+    runs-on: ubuntu-latest
+    needs: update
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```
Comment (p0deje) [inline .github/workflows/pin-browsers.yml]: Please use v4.

File: .github/workflows/pin-browsers.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      - name: Download patch
+        uses: actions/download-artifact@v2
```
Comment (p0deje) [inline .github/workflows/pin-browsers.yml]: Please use v4.

**Review by p0deje — APPROVED** [review]
This is really cool!

**Comment by titusfortner** [comment]
@p0deje you gave me the idea yesterday 😁 

And the old versions is because I was copy/pasting from chat gpt 😂 easy fix at least

**Review by diemol — APPROVED** [review]
Thank you! ❤️

**Pull request merged**